src/utils/actions.js

```diff
@@ -76,12 +80,13 @@ const baseIgActions = {
     context.flags['valid'] = validation.valid;
   },
   'submit-report': async context => {
-    await context.client.dest
-      .submit(context.reportingBundle)
+    await submitBundle(context.reportingBundle, context.client.dest)
       .then(
         result => {
+          console.log(result);
           if (result.status === StatusCodes.ACCEPTED || result.status === StatusCodes.OK) {
             context.flags['submitted'] = true;
+            debug(`Reporting Bundle submitted to ${context.client.dest}`);
```

Comment:
I like the `debug` lines better than the console.log/error but thinking forward a little bit to real use, can you add a bundle identifier of some kind to this line, so we can tell which bundle was submitted? Don't worry about adding new debug lines anywhere you haven't already, but where you do have them please add an ID for traceability